initial-promises/node-server/common/1.0/site.cf

```diff
@@ -83,6 +83,9 @@ bundle common g
                                               "curl/libcurl.dll",
                                               "curl/libssl32.dll",
                                               "curl/libeay32.dll",
+                                              "openssl/openssl.exe",
+                                              "openssl/libeay32.dll",
```

Comment:
I could try to identify against which OpenSSL version is what's in the curl directory and bundle this version in a directory containing both curl and openssl, or I could just replace curl's original binaries with the most recent ones if it breaks nothing.